ui/cap-react/src/style.less

```diff
@@ -88,5 +102,11 @@
 // Needed to make RJSF tooltip dialogs look the same as the rest
 .ant-tooltip .ant-tooltip-inner {
   border-radius: 2px;
-  font-family: "Titillium Web",
-}
\ No newline at end of file
+  font-family: "Titillium Web";
+}
+
+.__PublishedForm__ {
```

Comment:
In fact we shouldn't need this anymore since we will replace this view with the published version of fields (for TextArea it will just be a string instead of a disabled TextArea)